.github/workflows/CI.yml

```
@@ -0,0 +1,43 @@
+name: CI
+on:
+  - push
+  - pull_request
```

I prefer to avoid duplicated CI running when the PR is from the repo, so here limit `push` to `master` branch only.

```suggestion
on:
  create:
    tags:
  push:
    branches:
      - master
  pull_request:
  schedule:
    - cron: '20 00 1 * *'
```

Also, CI on tags and cron jobs sound worth to have.

---

I don't think `on: create` is a thing. I'll update the settings appropriately though